.github/workflows/publish-to-pypi.yml

```diff
@@ -0,0 +1,51 @@
+name: Publish package to PyPI
+
+on:
+  push:
+    tags:
+      - '*' # Push events to every tag not containing /
```

Comment:
Elaborate tag information bit more for better understanding

Reply:
like... what is a tag? or when it will push?

Reply:
ok good now